src/brick.c

```diff
@@ -567,8 +567,15 @@ static void do_unlink(struct pg_brick *brick, enum pg_side side, uint16_t index,
 		return;
 
 	unlink_notify(pair_edge, pg_flip_side(side), errp);
-	if (pg_error_is_set(errp))
-		return;
+	if (pg_error_is_set(errp)) {
+		if (!strcmp((*errp)->context.file, "src/tap.c") &&
```

Comment:
I think you can ignore all error here, not only the one that would come  from a tap